docs/news.rst

```diff
@@ -3,6 +3,20 @@
 Release notes
 =============
 
+
+.. _release-2.11.2:
+
+Scrapy 2.11.2
```

Comment:
I've not added release date here assuming this will be taken care during release